packages/components/psammead-grid/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Psammead Grid Changelog
+
+<!-- prettier-ignore -->
+| Version | Description |
+|---------|-------------|
+| 1.0.0-alpha.0 | [PR#x](https://github.com/bbc/psammead/pull/x) Initial package creation - Grid and GridItem exports. |
```

Comment:
PR number needs to be provided

Reply:
mentions ` Grid and GridItem exports.` however only exports `Grid`

Reply:
Done